### ISSUE TITLE
Check video / iframe on 404 page for accessibility

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6246]
From WGAG guidelines:

- iframes should be appropriately titled
- do not autoplay video
- enable youtube controls (allows autogenerated subtitles)
- provide audio transcript (maybe it is better to link to an external resource here, e.g. http://www.montypython.net/scripts/cheese.php)
